Add new API for adding local stores (#1549)
microsoft/appcenter-sdk-android: CHANGELOG.md

### App Center

> * **[Feature]** Add `Distribute.addStores(stores)` API for adding stores where should work in-app updates.

Reviewer: ```suggestion
* **[Feature]** Add `Distribute.addStores(stores)` API for adding local stores or installers which should allow in-app updates.
```
